course/include/isometry/matrix3.hpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * Vector library
+ * Author: Agustin Alba Chicar, 2019
+ * Author: Gerardo Puga, 2020
+ * Author: Steven Desvars, 2020
+ */
+
+#pragma once
+
+#include <isometry/vector3.hpp>
+#include <sstream>
+
+namespace ekumen {
+
+namespace math {
+
+class Matrix3 {
+ public:
+  /// Constructs a Matrix3.
+  /// @param row_1_ Values of the first row of the Matrix.
+  /// @param row_2_ Values of the first row of the Matrix.
+  /// @param row_3_ Values of the first row of the Matrix.
+  Matrix3(const std::initializer_list<double>& list);
```

Comment:
Three minor things:

 - I recommend to change the parameter `list` by `values` in order to be explicit what it contains.
 - There is only one `@param`.
 - You probably want to throw when the size of the list is different than 9

```suggestion
  /// Constructs a Matrix3.
  /// @param values Elements of the matrix. 
  /// Elements will fill the matrix's rows from top to bottom.
  ///
  /// @throw std::invalid_argument When `values`.size() is different than 9.
  Matrix3(const std::initializer_list<double>& values);
```